### PR TITLE
feat: Add information about 1.20.5 changes to internals to nms guide

### PR DESCRIPTION
**Pull request by Leguan16**
I wasn't sure if I should rewrite rather than only adding admonitions. If I should do it lmk.

### CODE REVIEW

File: docs/paper/dev/misc/internal-code.mdx [hunk 1]
```diff
@@ -65,6 +72,13 @@ it makes the code easier to understand.
 
 ### Mojang-Mapped Servers
 
+:::info[Mojang-mapped runtime as of 1.20.5]
+
+As of 1.20.5, Paper ships with a Mojang-mapped runtime instead of reobfuscating the server to Spigot mappings.
+For more information, see [here](./project-setup#Plugin-remapping) and [here](./userdev#1205).
```
Comment (powercasgamer) [inline docs/paper/dev/misc/internal-code.mdx:78]:
```suggestion
For more information, see [plugin-remapping](./project-setup#Plugin-remapping) and [userdev](./userdev#1205).
```

Just an idea

File: docs/paper/dev/misc/internal-code.mdx [hunk 2]
```diff
@@ -93,7 +107,6 @@ int dataVersion = Bukkit.getUnsafe().getDataVersion();
 
 :::danger[Parsing the version]
 
-You should not parse the version from the package name of classes. This is because CraftBukkit may not be relocated
-in the future.
+Parsing the version from the package name of classes is no longer possible as of 1.20.5 as Paper stopped relocating the CraftBukkit package. See [here](#reflection) for more information.
```
Comment (powercasgamer) [inline docs/paper/dev/misc/internal-code.mdx:110]:
```suggestion
Parsing the version from the package name of classes is no longer possible as of 1.20.5 as Paper stopped relocating the CraftBukkit package. See [reflection](#reflection) for more information.
```

Also just an idea

**Review by powercasgamer — CHANGES_REQUESTED** [review]

File: docs/paper/dev/misc/internal-code.mdx (hunk 1 above)
Comment (Leguan16) [reply docs/paper/dev/misc/internal-code.mdx]: altered the sentences a bit.

File: docs/paper/dev/misc/internal-code.mdx [hunk 3]
```diff
@@ -46,6 +46,13 @@ of Paper. This is unlikely to be the case in most production environments. This
 include the version. For example, `org.bukkit.craftbukkit.v1_20_R2.CraftServer` is the full class and package name
 for the CraftServer class in version 1.20.2. You can access these classes easily with some reflection utilities.
 
+:::warning[Removal of relocation in 1.20.5]
+
+As of 1.20.5 Paper removed the relocation of the CraftBukkit package into a versioned package.
```
Comment (zlataovce) [inline docs/paper/dev/misc/internal-code.mdx:51]:
```suggestion
As of 1.20.5, Paper removed the relocation of the CraftBukkit package into a versioned package.
```

File: docs/paper/dev/misc/internal-code.mdx [hunk 4]
```diff
@@ -93,7 +107,6 @@ int dataVersion = Bukkit.getUnsafe().getDataVersion();
 
 :::danger[Parsing the version]
 
-You should not parse the version from the package name of classes. This is because CraftBukkit may not be relocated
-in the future.
+Parsing the version from the package name of classes is no longer possible as of 1.20.5 as Paper stopped relocating the CraftBukkit package. See the [reflection](#reflection) section for more information.
```
Comment (zlataovce) [inline docs/paper/dev/misc/internal-code.mdx:110]:
```suggestion
Parsing the version from the package name of classes is no longer possible as of 1.20.5 as Paper stopped relocating the CraftBukkit package. See the [Reflection](#reflection) section for more information.
```

File: docs/paper/dev/misc/internal-code.mdx [hunk 5]
```diff
@@ -65,6 +72,13 @@ it makes the code easier to understand.
 
 ### Mojang-Mapped Servers
 
+:::info[Mojang-mapped runtime as of 1.20.5]
+
+As of 1.20.5, Paper ships with a Mojang-mapped runtime instead of reobfuscating the server to Spigot mappings.
+For more information, see the [plugin remapping](./project-setup#plugin-remapping) section and [userdev](./userdev#1205) documentation covering these changes.
```
Comment (zlataovce) [inline docs/paper/dev/misc/internal-code.mdx:78]:
```suggestion
For more information, see the [Plugin remapping](./project-setup#plugin-remapping) section and [userdev](./userdev#1205) documentation covering these changes.
```

File: docs/paper/dev/misc/internal-code.mdx (hunk 4 above)
Comment (Leguan16) [reply docs/paper/dev/misc/internal-code.mdx:110]: why exactly just here.

Comment (zlataovce) [reply docs/paper/dev/misc/internal-code.mdx:110]: it's the name of the section

Comment (Leguan16) [reply docs/paper/dev/misc/internal-code.mdx:110]: eh, its not really done at other places either.

File: docs/paper/dev/misc/internal-code.mdx [hunk 6]
```diff
@@ -46,6 +46,13 @@ of Paper. This is unlikely to be the case in most production environments. This
 include the version. For example, `org.bukkit.craftbukkit.v1_20_R2.CraftServer` is the full class and package name
 for the CraftServer class in version 1.20.2. You can access these classes easily with some reflection utilities.
 
+:::warning[Removal of relocation in 1.20.5]
+
+As of 1.20.5, Paper removed the relocation of the CraftBukkit package into a versioned package.
```
Comment (Machine-Maker) [inline docs/paper/dev/misc/internal-code.mdx:51]:
```suggestion
As of 1.20.5, the versioned relocation of the CraftBukkit package was removed.
```

File: docs/paper/dev/misc/internal-code.mdx [hunk 7]
```diff
@@ -46,6 +46,13 @@ of Paper. This is unlikely to be the case in most production environments. This
...
+This means that CraftBukkit packages will now always be located in `org.bukkit.craftbukkit` and no longer in `org.bukkit.craftbukkit.<version>`.
```
Comment (Machine-Maker) [inline docs/paper/dev/misc/internal-code.mdx:52]:
```suggestion
CraftBukkit packages are now located in `org.bukkit.craftbukkit` and not in `org.bukkit.craftbukkit.<version>`.
```

**Review by Machine-Maker — CHANGES_REQUESTED** [review]
I think making the wording more passive sounds better.

File: docs/paper/dev/misc/internal-code.mdx (hunk 7 above)
Comment (Leguan16) [reply docs/paper/dev/misc/internal-code.mdx]: addressed your requests. I just added an "and" to connect both sentences.

**Comment by MiniDigger** [comment]
>  Broken anchor on source page path = /paper/dev/internals:
     -> linking to ./userdev#1205 (resolved as: /paper/dev/userdev#1205)

**Comment by Leguan16** [comment]
ah yeah lol

**Comment by Leguan16** [comment]
> > Broken anchor on source page path = /paper/dev/internals:
> > -> linking to ./userdev#1205 (resolved as: /paper/dev/userdev#1205)

fixed xd

**Review by olijeffers0n — APPROVED** [review]

**Review by Machine-Maker — APPROVED** [review]

**Pull request merged**